Source/Scene/ModelExperimental/B3dmLoader.js

```diff
@@ -304,32 +304,23 @@ function createStructuralMetadata(loader, components) {
 
 // Recursive function to add the feature ID attribute to all primitives that have a feature ID vertex attribute.
 function processNode(node) {
-  if (!defined(node.children) && !defined(node.primitives)) {
-    return;
+  for (let i = 0; i < node.children.length; i++) {
```

Comment:
Can we extract the length variable here while we're at it? e.g. `const childrenLength = node.children.length`. just for consistency with the other changes in this PR